dist/scripts/windeployqt.ps1

```diff
@@ -22,24 +22,22 @@ if ($env:buildArch -ne 'Arm64') {
     Invoke-WebRequest -Uri $openSslDownloadUrl -OutFile openssl.zip
     7z x -y openssl.zip -o"openssl"
 
-    # Install approprate binaries for architecture
+    # Install appropriate binaries for architecture
     if ($env:buildArch -eq 'X86') {
-        copy openssl\$openSslSubfolder\x86\bin\libssl-$openSslFilenameVersion.dll bin\
-        copy openssl\$openSslSubfolder\x86\bin\libcrypto-$openSslFilenameVersion.dll bin\
+        copy "openssl\${openSslSubfolder}x86\bin\libssl-$openSslFilenameVersion.dll" bin
+        copy "openssl\${openSslSubfolder}x86\bin\libcrypto-$openSslFilenameVersion.dll" bin
     } else {
-        copy openssl\$openSslSubfolder\x64\bin\libssl-$openSslFilenameVersion-x64.dll bin\
-        copy openssl\$openSslSubfolder\x64\bin\libcrypto-$openSslFilenameVersion-x64.dll bin\
+        copy "openssl\${openSslSubfolder}x64\bin\libssl-$openSslFilenameVersion-x64.dll" bin
+        copy "openssl\${openSslSubfolder}x64\bin\libcrypto-$openSslFilenameVersion-x64.dll" bin
     }
 }
 
-if ($env:buildArch -eq 'Arm64') {
-    # Run windeployqt in cross-compilation mode
-    $qtHostPath = [System.IO.Path]::GetFullPath("$env:QT_ROOT_DIR\..\$((Split-Path -Path $env:QT_ROOT_DIR -Leaf) -replace '_arm64', '_64')")
-    & "$qtHostPath\bin\windeployqt" "--qmake=$env:QT_ROOT_DIR\bin\qmake.bat" --no-compiler-runtime bin\qView.exe
-} else {
-    # Run windeployqt which should be in path
-    windeployqt --no-compiler-runtime bin/qView.exe
-}
+# Run windeployqt
+$isCrossCompile = $env:buildArch -eq 'Arm64'
+$winDeployQt = $isCrossCompile ? "$(qmake -query QT_HOST_PREFIX)\bin\windeployqt" : "windeployqt"
+$argQtPaths = $isCrossCompile ? "--qtpaths=$env:QT_ROOT_DIR\bin\qtpaths.bat" : $null
+$argForceOpenSsl = $qtVersion -ge [version]'6.8' ? "--force-openssl" : $null
```

Comment:
Aside from some cleanup/refactoring in this area, the real change was passing `--force-openssl` since Qt no longer ships the OpenSSL backend plugin as of the 6.8 version of `windeployqt`.